src/SFA.DAS.Tools.Servicebus.Support.Application/Services/UserSessionService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.Extensions.Configuration;
```

Comment:
It looks like this logic would fit within the Query/Command classes as there's no real orchestration needed. So adding a Service I think is unecessary
I mean that the service just calls one Query/Command in each method so its an unecessary layer

Reply:
@DannyGB service also stores CurrentUserSession. That reduces number of requests we have to make to db